templates/patterns/featured_item/featured_item.ui_patterns.yml

```diff
@@ -25,14 +23,47 @@ featured_item:
         - "6 July 2015"
         - "Brussels"
     title:
-      type: "text"
+      type: "array"
       label: "Title"
-      description: "The title of the page."
-      preview: "Business, Economy, Euro"
+      description: "The title of the page. It must be an array compatible with Link component. If the variant is 'tile', only the label property is required."
+      preview:
+        label: "Business, Economy, Euro"
+        path: "http://example.com"
```

Comment:
We shouldn't change how our pattern behaves unless really necessary, this is after all our contract (API) towards developers using our theme.